### ISSUE TITLE
Watercooler - Q1 2017

### ISSUE THREAD

**Issue by Changaco** [liberapay/salon#98]
This is Liberapay's quarterly "what's up?" thread. Previous: #78.

**Comment by Changaco** [comment]
I've shut down https://sentry.changaco.oy.lc/ now that we have a sponsored account on https://sentry.io/. Next step: move Liberapay's email accounts away from my personal server as well.

**Comment by Changaco** [comment]
[New release: Bongos | MANGOPAY API Docs](https://docs.mangopay.com/blog/new-release-bongos)

> ### New functionality included in this release
> - Certain types of KYC docs can now be processed much more quickly
> - We've launched in private beta functionality that allows you to create an IBAN and BIC for a certain wallet and then send funds directly to the wallet (without having to create a payin bankwire beforehand)

**Comment by Changaco** [comment]
The next version of Diaspora* will allow podmins to easily add a Liberapay donation button: <https://github.com/diaspora/diaspora/pull/7290>.

**Comment by trebmuh** [comment]
awesome :+1:

**Comment by Changaco** [comment]
After a user sent an email asking about a payment issue, I ran `check-balances.py` and it found 4 mismatches.

It looks like we haven't been receiving MangoPay callbacks for a month, probably because I changed our CloudFlare settings a while ago to redirect all requests to HTTPS, and MangoPay has always rejected our certificate as being "not good enough".

I've added an exception on CloudFlare for `http://liberapay.com/callbacks/*`, and I've sent an email to MangoPay complaining that I hadn't received any alert from them about callback failures.

I've also fixed the balance mismatches of course, by manually redoing the callbacks.

**Comment by Changaco** [comment]
Someone posted feedback about the UI in French in <https://linuxfr.org/news/la-premiere-annee-de-liberapay#comment-1691419>. To me it's only a confirmation of things I already know, but maybe it'll motive someone else to roll up their sleeves and contribute.

**Comment by Changaco** [comment]
[MangoPay is lowering the 3DS threshold from 100€ to 50€](https://docs.mangopay.com/blog/anti-fraud-update-change-to-the-default-3ds-amount). That further decreases the benefit of bringing back automatic charges (see <https://github.com/liberapay/liberapay.com/issues/331>).

**Comment by Changaco** [comment]
[Open Source Guide - A community guide for open source creators.](https://opensource.guide/)

[Getting Paid for Open Source Work - Open Source Guide](https://opensource.guide/getting-paid/)

**Comment by Changaco** [comment]
[New release: Whale | MANGOPAY API Docs](https://docs.mangopay.com/blog/new-release-whale)

> ### New functionality included in this release
>
> - Add an automatic expiration for old payin bankwires (that are more than one month old)
> - We've rebuilt the KYC engine from the ground up to make it more powerful and intelligent - most noteably, you can now consult the amount of [cashin/cashout for a user](https://docs.mangopay.com/endpoints/v2.01/user-emoney#e895_view-a-users-emoney) to see how close to the KYC limits they are. More info about KYC is [here](https://mangopay.desk.com/customer/en/portal/articles/2502587-what-is-kyc-?b_id=10427).

**Comment by Changaco** [comment]
[Incident report on memory leak caused by Cloudflare parser bug](https://blog.cloudflare.com/incident-report-on-memory-leak-caused-by-cloudflare-parser-bug/)

TL;DR there is a very small possibility that some private information belonging to Liberapay users was leaked.

Freenode sent a global notice advising webchat users to change their passwords, I don't think we want to do the same.

**Comment by Changaco** [comment]
The user most likely to be affected is in fact me, because I usually log in several times per week, thus increasing the probability that the POST of my password was leaked and recorded. My account is also the most interesting because it has admin privilege, so I've changed its password.

**Comment by Changaco** [comment]
[New release: Pumpkin | MANGOPAY API Docs](https://docs.mangopay.com/blog/new-release-pumpkin)

> ### New functionality included in this release
>
> - EMoney balance for every users is now available on our dashboard and our API. More info about Emoney [here](https://docs.mangopay.com/endpoints/v2.01/user-emoney#e894_the-user-emoney-object)
> - Add a new filter on transaction reports on Fees amount.

**Comment by Changaco** [comment]
Feedback in French from the [Catalyst team](https://liberapay.com/collectifcatalyst) on the shortcomings of Liberapay: https://docs.google.com/document/d/1R13rK6uJSt0t1Gc0cCHRLTFnIN46z_EurQDiTYDYv9k/edit

**Comment by Changaco** [comment]
[New release: Toucan | MANGOPAY API Docs](https://docs.mangopay.com/blog/new-release-toucan)

> ### New functionality included in this release
> - You can now create multiple Dashboard accesses for your account ([SSO's](https://docs.mangopay.com/endpoints/v2.01/sso#e872_the-sso-object)), and set specific rights for each one with a [permission group](https://docs.mangopay.com/endpoints/v2.01/permission-groups#e905_the-permission-group-object)
> - We've added a new tool for our technical support team to better analyse client hook notifications

**Comment by Changaco** [comment]
With https://github.com/liberapay/liberapay.com/issues/177 finally closed, we no longer have any open issue tagged as "Critical". :-)

**Comment by Changaco** [comment]
https://github.com/gratipay/inside.gratipay.com/issues/1015#issuecomment-289831720

> Gratipay is now a GitHub integration! https://github.com/integrations/gratipay 🎉
